deposit/src/main/java/edu/unc/lib/deposit/CleanupDepositJob.java

```diff
@@ -215,32 +201,16 @@ private void deleteCleanupFiles(Model m) {
 			}
 		}
 	}
-	
-	private SharedStagingArea getStorageArea(URI stagingUri) {
-		if(!tagPattern.matches(stagingUri)) return null;
-
-		SharedStagingArea area = stages.findMatchingArea(stagingUri);
-		if (area == null) {
-			LOG.error("Cannot find staging area for URI: " + stagingUri.toString());
-			return null;
-		}
-		if (!area.isConnected()) {
-			stages.connect(area.getURI());
-		}
-		
-		return area;
-	}
 
-	public Stages getStages() {
-		return stages;
+	public void setStatusKeysExpireSeconds(int seconds) {
+		this.statusKeysExpireSeconds = seconds;
 	}
 
-	public void setStages(Stages stages) {
-		this.stages = stages;
+	public StagingPolicyManager getStagingManager() {
```

Comment:
Could we rename this method "getStagingPolicyManager()" just to make it clearer what it's returning?